src/components/link/link.ts

```diff
@@ -77,7 +77,7 @@ export class MLink extends ModulVue {
 
     private onClick(event): void {
         this.$el.blur();
-        if (this.isButton || this.disabled) {
+        if (this.isButton || this.disabled || this.isTextLink) {
```

Comment:
Tu devrais mettre  if (this.isButton || this.isTextLink || this.disabled ) {... juste plus clean pour la lisibilité